secret.go

```diff
@@ -51,6 +52,10 @@ func applySecret(ctx *cli.Context) error {
 		println(fmt.Sprintf("Secret type %s requires file path to create a valid filetype secret", secretType))
 		return nil
 	}
+	if secretName == "" {
+		println("Secret name cannot be empty")
```

Comment:
This error message can be more detailed, telling the user what to do. Something like:
"Secret name can not be empty. Please provide --secret-name <value> to proceed further..."

Right now it looks like as follows:
<img width="1141" alt="image" src="https://github.com/harness/harness-cli/assets/117125370/508a7be8-e09f-4867-bb54-3f3ea4fe1ce5">
Leaving the user hanging and not knowing how to proceed...